io_scene_godot/converters/utils.py

```diff
@@ -161,8 +161,9 @@ def to_mesh(self, preserve_vertex_groups=True,
                     except RuntimeError:
                         # Mesh must have n-gons
                         logging.warning(
-                            "Mesh had n-gons and had to be triangulated to "
-                            "calculate tangents; n-gons may look wrong."
+                            "Mesh %s had n-gons and had to be triangulated to "
```

Comment:
I think it's better to quote names, just in case someone uses a mesh name that could make the sentence sound confusing :slightly_smiling_face: 

```suggestion
                            "Mesh \"%s\" had n-gons and had to be triangulated to "
```